### DISCUSSION TITLE
@zeppos/zeus-cli@1.3.0-beta9 incorrectly scale images

### DISCUSSION THREAD

**Discussion by melianmiko** [zepp-health/zeppos-forum#210] [Developer Feedback]
Tried latest public-available beta version of CLI. And this version breaks image sizes in my projects which target to ZeppOS 1.0.
Icons should be 32x32 px, but this version of compiler scales them to approximately 24x24...

![image](https://github.com/zepp-health/zeppos-forum/assets/67851274/a3385d69-675c-4433-ba1d-5c8925210ebd)

**Comment by MyoungXUE** [comment]
@melianmiko 
Hi, can you provide us with the configuration of the targets in the Mini Program `app.json`, we need more clues to troubleshoot this issue, thanks for your cooperation.

**Reply by melianmiko** [reply]
https://github.com/melianmiko/ZeppOS-Tasks/blob/main/app.json

**Reply by MyoungXUE** [reply]
Thank you for your feedback, the CLI does have problems with the logic of image processing, for `targets` that do not fill in the `designWidth`, the scaling of the image will be inaccurate, we will release a new version as soon as possible to solve this problem!

At this stage, you can add the `designWidth` field to the target of each device in app.json to avoid this problem, the value can be set to the width of each device, which can be found at https://docs.zepp.com/docs/reference/related-resources/device-list/

**Reply by melianmiko** [reply]
Thanks for information, I've just downgraded back to stable version for now)

**Reply by htoooth** [reply]
@melianmiko We has fixed this issue. Please upgrade to latest beta version.

**Reply by melianmiko** [reply]
> We has fixed this issue. Please upgrade to latest beta version

Yep, `beta10` works fone. Thank you.
